### DISCUSSION TITLE
Allow the condense command to compact snowballs into snow blocks, and vice versa with /uncondense

### DISCUSSION THREAD

**Discussion by TheCashFire** [Zrips/CMI#6677] [Feature Requests]
A player of mine requested that snow balls be compactable into snow blocks at the appropriate ratio of 4:1, and uncondenseble at the same rate of 1:4. Small change, hope it passes. 

Thanks in advance!

**Comment by MrCrazys** [comment]
> A player of mine requested that snow balls be compactable into snow blocks at the appropriate ratio of 4:1, and uncondenseble at the same rate of 1:4. Small change, hope it passes.
> 
> Thanks in advance!

All you have to do is create a custom recipe for snow blocks to be crafted into snow blocks, then it will be possible both ways.

**Comment by mrfloris** [comment]
> A player of mine requested that snow balls be compactable into snow blocks at the appropriate ratio of 4:1, and uncondenseble at the same rate of 1:4. Small change, hope it passes.
> 
> Thanks in advance!

It would be cool if there's an extra parameter to not just condense but craft certain things. But I wouldn't want this automatically for everything. Each time I condense stone bricks, I wouldn't want to end up with an inventory of slabs, stairs, walls, etc. 

Technically a custom alias with hasitem: and the saveditems combo could be used to grab x snowballs and replace with y snowblocks or something. More info about specialized commands here: https://www.zrips.net/cmi/commands/specialized/

And like crazy said, the custom recipe is a nice gameplay solution as well.
